src/pages/notebooks/index.tsx

```diff
@@ -11,11 +11,7 @@ const NotebooksPage = () => {
     return null;
   }
 
-  return (
```

Comment:
Why do you edit this page ?

Reply:
Ooh that's a mistake. Let me get back the content

Reply:
Ooh that's a mistake. Let me get back the content